.teamcity/Gradle_Check/configurations/GradleBuildConfigurationDefaults.kt

```diff
@@ -150,7 +150,7 @@ fun BaseGradleBuildType.killProcessStepIfNecessary(stepName: String, os: Os = Os
             gradleWrapper {
                 name = stepName
                 executionMode = BuildStep.ExecutionMode.ALWAYS
-                tasks = "killExistingProcessesStartedByGradle"
+                tasks = "killExistingProcessesStartedByGradle -DpublishStrategy=publishOnFailure" // https://github.com/gradle/gradle-enterprise-conventions-plugin/pull/8
```

Comment:
Do you know what the difference between `tasks` and `gradleParams` means? Given the name, I would expect the system property go to `gradleParams`.

Reply:
They have no differences when merged into the final parameters, but yes, it SHOULD be in `gradleParams`.